task_sdk/src/airflow/sdk/api/client.py

```diff
@@ -157,6 +157,12 @@ def get(self, key: str) -> VariableResponse:
         resp = self.client.get(f"variables/{key}")
         return VariableResponse.model_validate_json(resp.read())
 
+    def put(self, msg):
+        """Put a variable through the API server."""
```

Comment:
Nit-ish: "put" is the implementation detail, the operation is "set"
```suggestion
    def set(self, msg):
        """Set an Airflow Variable via the API server."""
```

Reply:
Sure can do